assets/src/dashboard/app/index.js

```diff
@@ -53,6 +63,16 @@ const AppContent = () => {
     state: { currentPath },
   } = useContext(RouterContext);
 
+  useEffect(() => {
+    const dynamicPageTitle = ROUTE_TITLES[currentPath] || ROUTE_TITLES.DEFAULT;
+    window.document.title = sprintf(
+      /* translators: Admin screen title. 1: Admin screen name, 2: Network or site name. */
+      __('%1$s \u2039 %2$s \u2212 WordPress', 'web-stories'),
+      dynamicPageTitle,
+      ADMIN_TITLE
+    );
```

Comment:
@swissspidy is this more what you had in mind?

Reply:
Yay! :-)

Much better, thanks!

As mentioned, we could also pass the `'%1$s \u2039 %2$s \u2212 WordPress'` string via the config provider from the PHP side, so that we have less WordPress specific functionality within the app itself.

Reply:
I'll leave that choice up to you - but I think for the time being this is a great first iteration - there's no real logic getting decided that wasn't already up to the router and it keeps one directional data flow structure. We already have the info from the routerContext and the pageTitle from our nav set up so it's not too much work to configure the title string here - plus fewer pressing tickets for the workspace team (I forget the new pod name. 😄 ).